conans/cli/output.py

```diff
@@ -153,3 +153,8 @@ def _init_colors():
                 # TODO: check if colorama checks for stripping colors are enough
                 colorama.init()
             return True
+
+
+def cli_out_write(data, fg=None, bg=None, endline="\n"):
+    data = "{}{}{}{}{}".format(fg or '', bg or '', data, Style.RESET_ALL, endline)
+    sys.stderr.write(data)
```

Comment:
Not sure about the intention of this, if ``conan help`` output for example should go to stderr or to stdout? Is in this case the output considered the cli printing?

Reply:
I think this should be `sys.stdout`, not `stderr`

Reply:
Yes, this one should go to `sys.stdout ` this is the one that is used in the formatters of the commands and all the output should go to `stdout`.

Reply:
Fixed